SQF/dayz_code/actions/salvage_vehicle.sqf

```diff
@@ -57,7 +55,7 @@ if (_is6WheelType) then {
 
 		//get every damaged part no matter how tiny damage is!
 		_damagePercent = str(round(_damage * 100))+"% Damage";
-		if (_damage < 1 && {_damage > 0}) then {
+		if (_damage < 1) then {
```

Comment:
As vbawol mentioned [[here]](https://github.com/EpochModTeam/DayZ-Epoch/commit/3292d84b857f307571af9233dcfffe9d59a0b5e9#commitcomment-18424167) there are apparently some hitpoints that do not get damaged and allow infinite removal. I don't see any obvious issue with vehicle_getHitpoints or the way the salvage scripts get the hitpoints list, so I wouldn't be surprised if it is just a problem with certain vehicle configs. Until the problem is solved it is better to just leave this as is. It is more game breaking if players can get infinite parts from certain vehicles than if they can't salvage 0% damage ones.